app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -47,6 +50,11 @@ public class TabFragment extends Fragment {
     private RecyclerView recyclerView;
     private TransAdapter mAdapter1;
 
+    /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.